stretch/virtualbox/Vagrantfile

```diff
@@ -128,4 +128,6 @@ Vagrant.configure('2') do |config|
     config.vm.provision "shell", run: "always", env: {DISK_SIZE: grow_by.to_i},
                         path: 'https://raw.githubusercontent.com/subutai-io/packer/master/provisioning/en/subutai_disk.sh'
   end
+
+  config.vm.provision :blueprint
```

Comment:
Should we put a conditional around this to check the plugin version to protect against wrong plugin versions?